content/applications/sales/point_of_sale/overview/customer_notes.rst

```diff
@@ -0,0 +1,38 @@
+==============
+Customer notes
+==============
+
+**Customer notes** allow you to provide customers with further information on a specific product.
+For instance, you can give them cleaning and maintenance tips, etc. They can also be used to track a
+customer’s particular requests. For example, they do not want the product to be assembled, etc.
+
+Add customer notes
+==================
+
+You can add notes directly from an open :ref:`POS session <pos/start-session>`. To do so, click
+:guilabel:`Customer Notes` on the pad. Doing so opens a pop-up window on which you can add the
+desired content for the note.
+
+.. note::
+   Product notes from an :doc:`imported SO <../shop/sales_order>` are displayed identically in the
+   cart.
+
+.. image:: customer_notes/customer-notes-ui.png
+   :align: center
+   :alt: customer note button and notes (SO and POS session) on products in the cart
+
+Modify a customer note
+======================
+
+Select the product from the cart to modify an existing note and click :guilabel:`Customer Notes`.
+Doing so reopens the previous pop-up window. Alter the text and click :guilabel:`Add`.
+
+Receipts and invoices
+=====================
+
+The customer notes appear on customers' receipts and invoices similarly to how they appeared on the
+cart, meaning under the related product.
+
+.. image:: customer_notes/notes-receipt.png
```

Comment:
Not sure what purposes the notes serve if all they display is "Customer Note from SO". Maybe you should change it with something like "(This is where the customer note appears)"